lib/html/template/rose-bush/file_view.html

```diff
@@ -0,0 +1,24 @@
+{% if line_numbers|length == 0 -%}
+<p>No results found.</p>
+{% else -%}
+<!-- File Line Numbers -->
+<div class="span1 text-right">
+<pre id="filelinenumbers" class="prettyprint">
+{% for num in line_numbers -%}
+<span><a id="{{num}}" onclick="FileLoader.gotoLine('{{num}}')">{{num}}</a></span>
```

Comment:
Otherwise, it is often cleaner to inject the click functionality via JavaScript.